shared/js/smart-screen/scrollable.js

```diff
@@ -144,11 +144,13 @@
       var distance = Math.abs(this._getScrollOffset(newItem) -
                               this.translateX);
       var duration = (distance < 960) ? 0.3 : distance / 2000;
-
-      this.listElem.style.transitionDuration = duration + 's';
+      if (!this.listElem.classList.contains('reversed')) {
```

Comment:
Let's remove this clause since it would be removed right away in `endSlide()` in our case.